src/ResourceManager/Profile/ChangeLog.md

```diff
@@ -27,6 +27,8 @@
 * Add-AzureRmAccount
   * Added -MSI login for authenticationg using the credentials of the Managed Service Identity of the current VM / Service
   * Fixed KeyVault Authentication when logging in with user-provided access tokens
+* Add-AzureRmEnvironment and Set-AzureRmEnvironment
```

Comment:
@csuich2 @alexeldeib same comment about moving the lines you added to be under the `## Current Release` header